htdocs/comm/propal/class/propal.class.php

```diff
@@ -1455,6 +1455,12 @@ public function createFromClone(User $user, $socid = 0, $forceentity = null, $up
 							$line->subprice = $pu_ht;
 							$line->tva_tx = $tva_tx;
 							$line->remise_percent = $remise_percent;
+							// Update also cost price
```

Comment:
When we make a clone, the data of the new object must match the value of the src.